client/main.go

```diff
@@ -76,7 +76,8 @@ func (f AFLFuzzCommand) cmd() *exec.Cmd {
 	c := exec.Command(aflPath(),
 		"-o", "output",
 		"-i", "input",
-		"-S", f.fuzzerId)
+		"-S",
+		"--", "./target")
```

Comment:
This just wasn't working before, the target gets downloaded to a location called `target`. We'll want to figure out what to do with this properly and how we want to name things, but in the meantime this at least work.

Reply:
So my recollection that in master/slave (urgh, michal why you have to name things so badly) it inherits the command to run from the master process.

Reply:
Potentially it wasn't working because there wasn't a master preexisting? (That's super possible, maybe it was working on the bench because I left some master-was-running state lying around)